src/lib/fonts/shell-commands.rb

```diff
@@ -54,7 +59,7 @@ def self.fonts_config_file(file_id)
 
       cmd = "#{FONTS_CONFIG_CMD} --info"
       result = Yast::SCR.Execute(BASH_SCR_PATH, cmd)
-      file = result["stdout"].lines.select{|l| l =~ /#{file_id}/}[0].gsub(/.*: /, '').gsub(/\n/, '')
+      file = result["stdout"].lines.select{|l| l =~ /^[ ]*#{file_id}:/}[0].gsub(/.*: /, '').gsub(/\n/, '')
```

Comment:
if you want filter out whispaces, better way is `\s*` see e.g. for playing http://rubular.com/